### DISCUSSION TITLE
OPA deployment in kubernetes with nginx as Bundle server

### DISCUSSION THREAD

**Discussion by vishnupn** [open-policy-agent/community#149] [Community]
I am working on setting up a local kubernetes environment with OPA and nginx as bundle server. 
Bundle file kept in the source folder has to be added to ngnix pod on deployment.Any one has the  kubernetes deployment & service yaml file for  this set up?

**Comment by anderseknert** [comment]
You mean an example deployment for OPA? Here's an annotated deployment [example](https://github.com/anderseknert/opa-sidecar) you could use as a base for your own config.

**Reply by vishnupn** [reply]
Thanks for the reply.. I was looking for nginx yaml files with command to upload bundle file (tar gz) from local folder. I think bundle file has to be loaded to nginx on pod start
